source/quick-start.txt

```diff
@@ -50,6 +50,12 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. tip:: Download the JAR File
+   
+   You can directly download the JAR file for the {+driver-short+}
+   from the `Maven MongoDB Driver artifact entry
+   <https://mvnrepository.com/artifact/org.mongodb/mongodb-driver-sync/{+full-version+}>`__.
+
```

Comment:
Suggestion:
I think this could use similar wording as [this download mention](https://github.com/mongodb/docs-java/blob/master/source/fundamentals/data-formats/document-data-format-bson.txt#L89-L91):

"You can download the JAR file for the MongoDB Java Driver from the Maven repository."